lib/pacemaker/pcmk_status.c

```diff
@@ -107,6 +107,13 @@ pcmk__output_cluster_status(pcmk__output_t *out, stonith_t *stonith, cib_t *cib,
     data_set->priv = out;
     cluster_status(data_set);
 
+    if ((pcmk__our_nodename == NULL) && (cib->variant == cib_native)
+        && pcmk_is_set(show, pcmk_section_times)) {
+        // Currently used only in the times section
+        pcmk__query_node_name(out, 0, &pcmk__our_nodename, 0);
+        data_set->localhost = pcmk__our_nodename;
```

Comment:
This is a bug, here and in `pcmk_simulate.c`. `data_set->localhost` only gets set on the first call, but each call gets a fresh `data_set`. This made sense in this commit's original form, before we were using a static variable. Sanity-testing an update and then pushing.

Reply:
...one more time.